sdk/js/__tests__/Client.spec.js

```diff
@@ -476,6 +476,46 @@ describe('DevCycleClient tests', () => {
             expect(variable.value).toEqual(cachedConfig.variables.key.value)
             expect(onUpdate).toBeCalledTimes(1)
         })
+
+        it('should emit configUpdated event if debug user and etag matches', async () => {
+            const debugUserTestConfig = Object.assign({}, testConfig)
+            debugUserTestConfig.etag = 'test_etag'
+            debugUserTestConfig.sse = {
+                url: 'dvc_user',
+            }
+            client = createClientWithConfigImplementation(() => {
+                return Promise.resolve(debugUserTestConfig)
+            })
+
+            const onUpdate = jest.fn()
+            client.subscribe('configUpdated', onUpdate)
+            await client.onClientInitialized()
+            // trigger 2nd config call to check if etag matches
+            await client.identifyUser({
+                user_id: 'debugUser',
+            })
+            expect(onUpdate).toBeCalledTimes(2)
```

Comment:
could it make more sense if the subscribe is after `onClientInitialized` and so the test would show 1 update happened vs none instead of 2 vs 1

Reply:
Probably doesnt hurt to check that its getting called when there's no `oldConfig` yet as well

Reply:
just got confused for a second, maybe a comment that explains that? could just be me 😅